src/enums.ts

```diff
@@ -138,6 +138,12 @@ export enum OrderPaymentMethod {
     IN_PERSON = 'IN_PERSON',
 }
 
+export enum OrderModifierChoiceType {
```

Comment:
did you make sure to delete the one in the file then?

Reply:
The last transpile did not work because of the error so there is no duplicate stored in this build enums file